cmd/exit.go

```diff
@@ -14,10 +14,14 @@ const (
 	// 5: CodeNotFound
 	CodeFailedDetect           = 100
 	CodeFailedDetectWithErrors = 101
-	CodeFailedBuild            = 7
+	CodeDetectError            = 102
+	CodeAnalyzeError           = 200
+	CodeRestoreError           = 300
```

Comment:
```suggestion
	CodeAnalyzeError           = 202
	CodeRestoreError           = 302
```
Maybe this is super OCD but given that the generic detect error is `102` I like using `x02` for the other phases.